evalml/tests/component_tests/test_standard_scaler.py

```diff
@@ -1,32 +1,51 @@
 import pandas as pd
-import pytest
-import woodwork as ww
-from woodwork.logical_types import Boolean, Categorical, Double, Integer
+from pandas.testing import assert_frame_equal
 
 from evalml.pipelines.components import StandardScaler
 
 
-@pytest.mark.parametrize(
-    "X_df",
-    [
-        pd.DataFrame(pd.Series([1, 2, 3], dtype="Int64")),
-        pd.DataFrame(pd.Series([1.0, 2.0, 3.0], dtype="float")),
-        pd.DataFrame(pd.Series([True, False, True], dtype="boolean")),
-    ],
-)
-def test_standard_scaler_woodwork_custom_overrides_returned_by_components(X_df):
-    y = pd.Series([1, 2, 1])
-    override_types = [Integer, Double, Categorical, Boolean]
-    for logical_type in override_types:
-        try:
-            X_df.ww.init(logical_types={0: logical_type})
-        except ww.exceptions.TypeConversionError:
-            continue
-
-        std_scaler = StandardScaler()
-        std_scaler.fit(X_df, y)
-        transformed = std_scaler.transform(X_df, y)
-        assert isinstance(transformed, pd.DataFrame)
-        assert {k: type(v) for k, v in transformed.ww.logical_types.items()} == {
-            0: Double,
-        }
+def test_standard_scaler_applies_to_numeric_columns_only():
```

Comment:
This is only a nit as it relates to my current work with hardening our tests against woodwork changes - using pytest fixtures for tests means we have all our woodwork typing consolidated and have to change fewer things should we need to tweak types for compatibility reasons. Plus, it should reduce the number of lines of code here.
I think either the `get_test_data_from_configuration` or the `imputer_test_data` fixture should have the same column types you're looking for.